*** tests/A-12-03-M-ok.out	Fri Oct 14 17:14:01 2022
--- tests/A-12-03-M-ok.outhyp	Mon Oct 31 22:25:08 2022
***************
*** 53,59 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: VOICE|1|969004|969001|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 53,59 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: VOICE|0|BASIC|969004|cli002|BUSY|0|0|BASIC|969001|cli001|IDLE|0|0|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
