*** tests/A-14-02-M-ok.out	Fri Oct 21 12:17:06 2022
--- tests/A-14-02-M-ok.outhyp	Mon Oct 31 22:25:10 2022
***************
*** 34,40 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: VOICE|1|969001|969011|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 34,40 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: VOICE|0|BASIC|969001|cli001|BUSY|0|0|FANCY|969011|cli001|IDLE|0|0|0|0|ONGOING
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 56,62 ****
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
! FANCY|969011|cli001|BUSY|0|0
  FANCY|969013|cli001|SILENCE|0|0
  FANCY|969014|cli002|IDLE|0|0
  FANCY|969015|cli003|IDLE|0|0
--- 56,62 ----
  BASIC|969006|cli002|IDLE|0|0
  BASIC|969007|cli003|IDLE|0|0
  BASIC|969008|cli003|SILENCE|0|0
! FANCY|969011|cli001|IDLE|0|0
  FANCY|969013|cli001|SILENCE|0|0
  FANCY|969014|cli002|IDLE|0|0
  FANCY|969015|cli003|IDLE|0|0
***************
*** 73,78 ****
--- 73,80 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
