*** tests/A-15-05-M-ok.out	Mon Oct 24 17:15:43 2022
--- tests/A-15-05-M-ok.outhyp	Mon Oct 31 22:25:10 2022
***************
*** 69,76 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: VIDEO|1|969011|969014|30|900|FINISHED
! CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
  3 - Mostrar comunicações recebidas por um cliente
--- 69,75 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
  3 - Mostrar comunicações recebidas por um cliente
